examples/next/pages/ui/components/authenticator/sign-in-with-phone/index.page.tsx

```diff
@@ -32,10 +33,13 @@ const formFields = {
 function App({ user, signOut }) {
   return (
     <>
-      <h2>Welcome, {user.attributes?.phone_number}</h2>
+      <h2>Welcome, {user?.attributes?.phone_number}</h2>
```

Comment:
user shouldn't be undefined here, right?

Reply:
In some cases while logging out this would throw an error, just added this to avoid the error for now.

Reply:
Can we add a @todo-migration reminder to revert? Feel like this could hide issues.